Tokenomics_Network_Report_Template.md

```diff
@@ -0,0 +1,122 @@
+| Number of members |              |              |          |
+
+## 2.0 Tokenomics
+### 2.1 Token generation breakdown
+| Property                    | This Session | All Sessions | % Change |
+|-----------------------------|--------------|--------------|----------|
+| Total Tokens Minted        |              |              |          |
+| Validator Role        |              |              |          |
+| Storage Role       |              |              |          |
+| Council Role |              |              |          |
+
+### 2.2 Mints 
+| Property                    | This Session | All Sessions | % Change |
+|-----------------------------|--------------|--------------|----------|
+| Council Mint Capacity        |              |              |          |
+| Council Mint Total Minted       |              |              |          |
+| Curator Mint Capacity |              |              |          |
+| Curator Mint Total Minted |              |              |          |
+
+## 3.0 Council
+* Council session #: XX
+* Number of council members:
+* Total number of proposals:
+### 3.1 Elections
+| Property                    | This Session | All Sessions | % Change |
+|-----------------------------|--------------|--------------|----------|
+| Total Applicants       |              |              |          |
+| Total Applicant Stake       |              |              |          |
+| Total Votes       |              |              |          |
+| Avg Votes per Applicant       |              |              |          |
+
+### 3.2 Proposals
+| Proposal Type                           | # of proposals during this session | Total number of proposal type |
+|-----------------------------------------|------------------------------------|-------------------------------|
+| Text                                    |                                    |                               |
+| Runtime Upgrade                         |                                    |                               |
+| Set Election Parameters                 |                                    |                               |
+| Spending                                |                                    |                               |
+| Set Lead                                |                                    |                               |
+| Set Content Working Group Mint Capacity |                                    |                               |
+| Evict Storage Provider                  |                                    |                               |
+| Set Validator Count                     |                                    |                               |
+| Set Storage Role Parameters             |                                    |                               |
+* Average time for proposal vote success:
+* Average overall time for proposal vote success:
+
+## 4 Roles
+### 4.1 Validator Information
```

Comment:
As the reward here is a function of the `slotStake`, not the average stake, consider taking that into account?
Also, this role sees more slashing than others, so I think that could make sense to add as well :)